frontend/src/components/funding/FundingAgreementsTab.vue

```diff
@@ -84,19 +116,21 @@ export default {
         this.fundingAgreements?.sort((a, b) => b.priority - a.priority) // FA Signature Pending status at the top
       } catch (error) {
         this.setFailureAlert('Failed to load funding agreements', error)
-      } finally {
```

Comment:
there is a Search card in this view. if we remove loading = false from this function, the screen will stuck at loading when the user click search button. I think we should keep this finally part